core/transaction/pool/pool.go

```diff
@@ -160,19 +158,25 @@ func (tp *TxnPool) removeTransaction(txn *Transaction) {
 }
 
 //check and add to utxo list pool
-func (tp *TxnPool) apendToUTXOPool(txn *Transaction) (bool, *Transaction) {
+func (tp *TxnPool) checkReferencedUTXO(txn *Transaction) error {
```

Comment:
The name suggests it's only checking, but it's actually doing both checking and appending, which may cause confusion.

Reply:
@yilunzhang Fixed. Renamed it to checkAndAddReferencedUTXO.